lib/results_validators/persons_validator.rb

```diff
@@ -60,7 +60,7 @@ def self.dob_validations(dob, competition_id = nil, **message_args)
       validation_issues
     end
 
-    def self.name_validations(name, competition_id = nil, **message_args)
+    def self.name_validations(name, competition_id = nil, **_message_args)
```

Comment:
Curious that this doesn't seem to be covered by `AllowUnusedKeywordArguments`... Have you double-checked? Or might this be a leftover from the time before you introduced that config option I mentioned?

Reply:
sadly no